From ca97ce2562ac7510e48e5c531eb0a3264f6c1144 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Fri, 14 Feb 2025 11:43:17 +0100
Subject: [PATCH] ARM: configs: stm32mp2: Enable smc_watchdog for aarch32

Watchdog kernel support is enabled by op-tee. This is needed in case the
iwdg1 mapped to arm_wdt watchdog was started in tf-a.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: Ic982ffe3d6763a521aae47fa2df7961befc3ed86
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/437229
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Antonio Maria BORNEO <antonio.borneo@foss.st.com>
---
 arch/arm/configs/mp2-fragment-04-multiv7-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/mp2-fragment-04-multiv7-addons.config b/arch/arm/configs/mp2-fragment-04-multiv7-addons.config
index e2c0a166c9ed..f5bc34dfe24e 100644
--- a/arch/arm/configs/mp2-fragment-04-multiv7-addons.config
+++ b/arch/arm/configs/mp2-fragment-04-multiv7-addons.config
@@ -1,2 +1,3 @@
 CONFIG_ARM_LPAE=y
+CONFIG_ARM_SMC_WATCHDOG=y
 
-- 
2.50.1

